charts/super-agent/values.yaml

```diff
@@ -94,71 +94,84 @@ super-agent-deployment:
-        #   endpoint: https://opamp.service.newrelic.com/v1/opamp
-        #   endpoint: https://opamp.service.eu.newrelic.com/v1/opamp
-
-        # -- This option enables a status server that can be useful for troubleshooting.
-        # -- Port-forward it `$ kubectl port-forward pod/{pod-name} 51200:51200`
-        # -- And query it as `$ curl localhost:51200/status`
-        server:
-          enabled: true
-        #   port: 51200
-
-
-    # -- Values that the fleet is going to have in the deployment.
-    # @default -- See `values.yaml` for examples
+      # -- It you are a PowerUser, you can use this to override the configuration that has been created automatically by the chart.
+      # This configuration here will be **MERGED** with the configuration specified above.
+      # If you need to have you own configuration, disabled the creation of this configMap and create your own.
+      content: {}
+
+    # -- Values that the fleet is going to have in the deployment. If empty, chart will automatically add `newrelic/io.opentelemetry.collector` subagent.
+    # On the other hand, if populated the list of agent created is the one specified overwriting the default.
+    # @default -- `newrelic/io.opentelemetry.collector` (See `values.yaml`)
     subAgents:
-      # The values of the fleet depends on the deployment itself. Each subagent has a different set of variables so you have to go to the subagent documentation
-      # find the configuration needed for the subagent.
-      #
-      open-telemetry:
-        content:
-          # chart_version: "0.4.0"
-          chart_values:
-            licenseKey: ${nr-env:NR_LICENSE_KEY}
-            cluster: ${nr-env:NR_CLUSTER_NAME}
-            # TODO the following values are set twice in the config, we have to add some logic to improve UX either in the chart or in the agentType
-            # customSecretName: ""
-            # customSecretLicenseKey: ""
+    # The values of the fleet depends on the deployment itself. Each subagent has a different set of variables so you have to go to the subagent documentation
+    # find the configuration needed for the subagent.
+    #
+    # The example below, open-telemetry, is enabled by default if no subagent is set up.
+    # open-telemetry:
+    #   type: newrelic/io.opentelemetry.collector:0.2.0
+    #   content:
+    #     chart_values:
+    #       global:
+    #         licenseKey: ${nr-env:NR_LICENSE_KEY}
+    #         cluster: ${nr-env:NR_CLUSTER_NAME}
+    #         nrStaging: ${nr-env:NR_STAGING}
+    #         verboseLog: ${nr-env:NR_VERBOSE}
+    #         region: ${nr-env:NR_REGION}
+    #         # you can set here modifications to the open telemetry chart
+
+    opamp:
+      # -- Enables or disables the auth against fleet control. It implies to disable any fleet communication and running the agent
+      # in stand alone mode where only the agents specified on `.config.subAgents` will be launched.
+      enabled: true
+
+    auth:
```

Comment:
i guess this was already discussed but shouldn't be auth inside opamp config?

Reply:
No, it was not discussed. the discuss was that the `enabled` part should be in `opamp` instead of `auth`: https://github.com/newrelic/helm-charts/pull/1466#discussion_r1749977943

I am a bit reluctant to think that `auth` is going to be used only for `opamp` given the future I see about downloading packages and such.

This is just a feeling, if you don't feel the same way I have no issues on changing it.

Reply:
i would perhaps put it inside opamp just to be consistent with the SA config but not an strong opinion.

Reply:
Done in [#d598ded](https://github.com/newrelic/helm-charts/pull/1466/commits/d598dedf47d32be2ea9e055031faf859b75386fe)